image_registry.yaml

```diff
@@ -1,342 +1,817 @@
-  tag_regex: '^1\.\d+\.\d+$'
+  default_tag: 1.24.2
   tags:
-    - 1.23.0
-    - 1.24.0
+    - 1.24.2
     - 1.24.1
+    - 1.24.0
+    - 1.23.0
+  tag_regex: ^1\.\d+\.\d+$
 forgejo:
-  default_tag: 1.21.11-2
   name: codeberg.org/forgejo/forgejo
-  tag_regex: '^\d+\.\d+\.\d+-\d+$'
+  default_tag: 1.21.11-2
   tags:
     - 1.21.11-2
+  tag_regex: ^\d+\.\d+\.\d+-\d+$
 frrouting_frr:
-  default_tag: 9.1.0
   name: quay.io/frrouting/frr
-  tag_regex: '^\d+\.\d+\.\d+$'
-  tags:
+  default_tag: 10.3.1
+  tags:
+    - 10.3.1
+    - 10.3.0
+    - 10.2.3
+    - 10.2.2
+    - 10.2.1
+    - 10.2.0
+    - 10.1.3
+    - 10.1.2
+    - 10.1.1
+    - 10.1.0
+    - 10.0.3
+    - 10.0.2
+    - 10.0.1
+    - 10.0.0
+    - 9.1.3
+    - 9.1.2
+    - 9.1.1
     - 9.1.0
+  tag_regex: ^\d+\.\d+\.\d+$
 grafana:
-  default_tag: 11.3.1
   name: docker.io/grafana/grafana
-  tag_regex: '^\d+\.\d+\.\d+$'
-  tags:
-    - 10.4.5
-    - 11.2.0
+  default_tag: 12.0.1
+  tags:
+    - 12.0.1
+    - 12.0.0
+    - 11.6.2
+    - 11.6.1
+    - 11.6.0
+    - 11.5.5
+    - 11.5.4
+    - 11.5.3
+    - 11.5.2
+    - 11.5.1
+    - 11.5.0
+    - 11.4.5
+    - 11.4.4
+    - 11.4.3
+    - 11.4.2
+    - 11.4.1
+    - 11.4.0
+    - 11.3.7
+    - 11.3.6
+    - 11.3.5
+    - 11.3.4
+    - 11.3.3
+    - 11.3.2
     - 11.3.1
+    - 11.2.0
+    - 10.4.5
+  tag_regex: ^\d+\.\d+\.\d+$
 istio_pilot:
-  default_tag: 1.24.5-distroless
   name: docker.io/istio/pilot
-  tag_regex: '^\d+\.\d+\.\d+-distroless$'
-  tags:
-    - 1.22.3-distroless
-    - 1.23.1-distroless
-    - 1.23.2-distroless
-    - 1.23.3-distroless
+  default_tag: 1.26.1-distroless
+  tags:
+    - 1.26.1-distroless
+    - 1.26.0-distroless
+    - 1.25.3-distroless
+    - 1.25.2-distroless
+    - 1.25.1-distroless
+    - 1.25.0-distroless
+    - 1.24.6-distroless
     - 1.24.5-distroless
+    - 1.23.3-distroless
+    - 1.23.2-distroless
+    - 1.23.1-distroless
+    - 1.22.3-distroless
+  tag_regex: ^\d+\.\d+\.\d+-distroless$
 istio_proxy:
-  default_tag: 1.24.5-distroless
   name: docker.io/istio/proxyv2
-  tag_regex: '^\d+\.\d+\.\d+-distroless$'
-  tags:
-    - 1.22.3-distroless
-    - 1.23.1-distroless
-    - 1.23.2-distroless
-    - 1.23.3-distroless
+  default_tag: 1.26.1-distroless
+  tags:
+    - 1.26.1-distroless
+    - 1.26.0-distroless
+    - 1.25.3-distroless
+    - 1.25.2-distroless
+    - 1.25.1-distroless
+    - 1.25.0-distroless
+    - 1.24.6-distroless
     - 1.24.5-distroless
+    - 1.23.3-distroless
+    - 1.23.2-distroless
+    - 1.23.1-distroless
+    - 1.22.3-distroless
+  tag_regex: ^\d+\.\d+\.\d+-distroless$
 jupyter_datascience_lab:
-  default_tag: lab-4.4.0
   name: quay.io/jupyter/datascience-notebook
+  default_tag: lab-4.4.3
   tags:
+    - lab-4.4.3
+    - lab-4.4.2
+    - lab-4.4.1
     - lab-4.4.0
-  tag_regex: '^lab-\d+\.\d+\.\d+$'
+  tag_regex: ^lab-\d+\.\d+\.\d+$
 karpenter:
-  default_tag: 1.2.0
   name: public.ecr.aws/karpenter/controller
-  tags:
-    - 0.37.0
-    - 1.0.8
+  default_tag: 1.5.0
```

Comment:
This and AWS LB Controller are the scariest. They aren't tested and also have been difficult to upgrade historically.